src/deriveLinkDefaults.ts

```diff
@@ -158,9 +158,17 @@ export function initialLoader(datasets: ILinkConfig['datasets']) {
     return Promise.all(resolvedDataSets).then((data) => {
       const cleaned = data.filter((d): d is DataSet => d != null);
       cleaned.forEach((d) => {
-        if (d.params && !Array.isArray(d.params) && d.params._endpoint && d.params.regions) {
+        if (d.params && !Array.isArray(d.params) && d.params._endpoint) {
```

Comment:
Should we preserve the existing handling when `d.params.regions` is present? ([example](https://delphi.cmu.edu/epivis/#eyJjaGFydCI6eyJ2aWV3cG9ydCI6WzcyOTI5Ni41LC0wLjg3NTMyMjQ2MjcwOTk1NDUsNzM5MDk3LjUsOC45NDIzMDU4MzQ3MDg2NzhdLCJzaG93UG9pbnRzIjpmYWxzZX0sImRhdGFzZXRzIjpbeyJjb2xvciI6IiM0YTNjOTAiLCJ0aXRsZSI6IndpbGkiLCJwYXJhbXMiOnsiX2VuZHBvaW50IjoiZmx1dmlldyIsInJlZ2lvbnMiOiJuYXQifX1dfQ==))

This part is up to you, but i think you should be able to pad the code block with these block comments instead of annotating each of the 3-4 individual line disable comments:
```js
if (d.params && ...) {
  /* eslint-disable @typescript-eslint/restrict-template-expressions */
  INSERT_BLOCK_HERE;
  /* eslint-enable @typescript-eslint/restrict-template-expressions */
}
```